package.json

```diff
@@ -47,69 +47,45 @@
   "devDependencies": {
     "@chainsafe/eslint-plugin-node": "^11.2.3",
     "@dapplion/benchmark": "^0.2.4",
-    "@types/chai": "^4.3.6",
-    "@types/chai-as-promised": "^7.1.6",
-    "@types/mocha": "^10.0.1",
+    "@types/mocha": "^10.0.6",
```

Comment:
Interesting! Does vitest use the mocha types for compatibility or something?

Reply:
Our performance tests still use the mocha internally, so why we need those types.